build_tools/provision_machine.py

```diff
@@ -1,4 +1,29 @@
 #!/usr/bin/env python
+"""provision_machine.py
+
+This script  helps CI workflows, developers and testing suites easily provision TheRock to their environment.
```

Comment:
nit: remove extra space

```suggestion
This script helps CI workflows, developers and testing suites easily provision TheRock to their environment.
```